javascript-forms/form-events.js

```diff
@@ -0,0 +1,29 @@
+function handleFocus(event) {
+  console.log("'Focus' event was fired.");
+  console.log('event.target.name:', event.target.name);
+}
+
+function handleBlur(event) {
+  console.log("'Blur' event was fired.");
+  console.log('event.target.name:', event.target.name);
+}
+
+function handleInput(event) {
+  console.log('value of', event.target.name + ':', event.target.value);
+}
+
+var $userName = document.forms[0].elements[0];
+var $userEmail = document.forms[0].elements[1];
```

Comment:
You should still use `querySelector` here, this may break if another form is ever added to the page